docker/package/packages.py

```diff
@@ -17,7 +17,7 @@
 networks = {
     "mainnet": "mainnet",
     "ghostnet": "ghostnet",
-    "limanet": "https://teztnets.xyz/limanet",
+    "limanet": "limanet",
     "kathmandunet": "kathmandunet",
 }
 networks_protos = {
```

Comment:
oh, actually, since ghostnet will also switch to lima in the future (sooner than mainnet, in fact) let's also add the lima baker to it for the same reason